### PR TITLE
[Gameboard] Initializing an empty gameboard!!!

### PR DESCRIPTION
**Pull request by pranathip**
# Summary
Initializing an empty gameboard to start the game. There is no game logic yet, that will be addressed in the next diff of the stack.

# Images

### CODE REVIEW

**Comment by pranathip** [comment]
Current dependencies on/for this PR:
* main
  * **PR #21** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/21" target="_blank">Graphite</a>  👈
    * **PR #22** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/22" target="_blank">Graphite</a> 
      * **PR #23** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/23" target="_blank">Graphite</a> 
        * **PR #24** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/24" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/21?utm_source=stack-comment).

**Comment by pranasaurusrex** [comment]
Comment

File: src/Gameboard.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    [0, 0, 0, 0],
+    [0, 0, 0, 0]
+  ];
+  
+  const hasValue = (board, value) => {
+    for (let i = 0; i < board.length; i++) {
+      for (let j = 0; j < board[i].length; j++) {
+        if (board[i][j] === value) {
+          return true;
+        }
```
Comment (pranasaurusrex) [inline src/Gameboard.js:13]: This is a comment

File: src/Gameboard.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    return !hasValue(board, 0);
+  };
+  
+  const getRandomPosition = () => {
+    const rowPosition = Math.floor(Math.random() * 4);
+  const colPosition = Math.floor(Math.random() * 4);
+  return [rowPosition, colPosition];
+};
+
+export const generateRandom = (board) => {
```
Comment (pranasaurusrex) [inline src/Gameboard.js:29]: This is great!

**Comment by pranathip** [comment]
@pranathip merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Avocadoo-Corp/pranathi-2048-react-app/21).

**Pull request merged**